db/fixtures/ae_datastore/ManageIQ/Deployment/ContainerProvider/System/StateMachine/Deployment.class/__methods__/add_provider.rb

```diff
@@ -1,24 +1,12 @@
-require 'net/ssh'
-
 DEPLOYMENT_TYPES = {
-  :origin            => "ManageIQ::Providers::Openshift::ContainerManager",
-  :enterprise        => "ManageIQ::Providers::OpenshiftEnterprise::ContainerManager",
-  :atomic            => "ManageIQ::Providers::Atomic::ContainerManager",
-  :atomic_enterprise => "ManageIQ::Providers::AtomicEnterprise::ContainerManager"
+  :origin               => "ManageIQ::Providers::Openshift::ContainerManager",
+  :openshift_enterprise => "ManageIQ::Providers::OpenshiftEnterprise::ContainerManager",
+  :atomic               => "ManageIQ::Providers::Atomic::ContainerManager",
+  :atomic_enterprise    => "ManageIQ::Providers::AtomicEnterprise::ContainerManager"
```

Comment:
Those aren't supported by us yet and can't be activated, just placed it here for future use, should I remove?